README.md

```diff
@@ -15,36 +15,36 @@ and is available for use with the following clients.
 ### Completion for Liberty server configuration files
 * Completion for Liberty properties and values 
 
-![Completion feature for Liberty properties in bootstrap.properties](./docs/images/property-completion.png) 
-![Completion feature for Liberty property values in bootstrap.properties](./docs/images/property-value-completion.png)
+![Screenshot of Liberty property name suggestions in a bootstrap.properties file](./docs/images/property-completion.png) 
```

Comment:
Should we follow this pattern to have a concise title for most users of the document and a detailed description in the alt text for those users that need it?

```
![Alt text here](/path/to/img.jpg “image title”)
```

Reply:
I think that's a good idea